src/evm/decoder.rs

```diff
@@ -52,6 +67,10 @@ pub(super) struct TychoStreamDecoder {
     inclusion_filters: HashMap<String, FilterFn>,
 }
 
+/// Decodes incoming raw messages.
+///
+/// This struct decodes incoming messages of type `FeedMessage` and converts it into the
+/// `BlockUpdate`struct
```

Comment:
Docs on a impl?

Reply:
you literally asked for this on the previous review 😂 see [here](https://github.com/propeller-heads/tycho-simulation/pull/98/files/58a469f0da2750e677bf8660a03f2e34219ee650#r1884988796). I'm removing it then